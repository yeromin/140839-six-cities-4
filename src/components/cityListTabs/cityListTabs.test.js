import React from 'react';
import renderer from 'react-test-renderer';
import {CityListTabs} from './CityListTabs.jsx';

const currentCity = `Amsterdam`;

describe(`CityListTabs_test`, () => {
  it(`Component CityListTabs should render the proper Cities Tabs in Header`, () => {
    const tree = renderer.create(
        <CityListTabs
          currentCity={currentCity}
          handleCityClick={() => {}}
        />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
