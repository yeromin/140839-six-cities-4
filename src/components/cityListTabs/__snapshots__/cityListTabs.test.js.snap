// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CityListTabs_test Component CityListTabs should render the proper Cities Tabs in Header 1`] = `
<div
  className="tabs"
>
  <section
    className="locations container"
  >
    <ul
      className="locations__list tabs__list"
    >
      <li
        className="locations__item"
      >
        <a
          className="locations__item-link
                  tabs__item
                  "
          href="#"
          onClick={[Function]}
        >
          <span>
            Paris
          </span>
        </a>
      </li>
      <li
        className="locations__item"
      >
        <a
          className="locations__item-link
                  tabs__item
                  "
          href="#"
          onClick={[Function]}
        >
          <span>
            Cologne
          </span>
        </a>
      </li>
      <li
        className="locations__item"
      >
        <a
          className="locations__item-link
                  tabs__item
                  "
          href="#"
          onClick={[Function]}
        >
          <span>
            Brussels
          </span>
        </a>
      </li>
      <li
        className="locations__item"
      >
        <a
          className="locations__item-link
                  tabs__item
                  tabs__item--active"
          href="#"
          onClick={[Function]}
        >
          <span>
            Amsterdam
          </span>
        </a>
      </li>
      <li
        className="locations__item"
      >
        <a
          className="locations__item-link
                  tabs__item
                  "
          href="#"
          onClick={[Function]}
        >
          <span>
            Hamburg
          </span>
        </a>
      </li>
      <li
        className="locations__item"
      >
        <a
          className="locations__item-link
                  tabs__item
                  "
          href="#"
          onClick={[Function]}
        >
          <span>
            Dusseldorf
          </span>
        </a>
      </li>
    </ul>
  </section>
</div>
`;
